rules/mapping_json.go

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
These should be `2018` for the new files added.

Reply:
O yeah. This was a shameless copy paste, I think I need to update this in all the files.

Reply:
For all the new files yeah? This should be the date where the file was/is added, so you only need to update it for the new files you added.